### ISSUE TITLE
Check what Plugins we are missing for feature parity with Leaflet

### ISSUE THREAD

**Issue by essential-randomness** [FujoWebDev/astrolabe#49]
1. Go to [leaflet and open a note](https://leaflet.pub/)
2. Figure out what they have that we might also want
